internal/pkg/agent/application/application.go

```diff
@@ -305,3 +307,70 @@ func mergeFleetConfig(ctx context.Context, rawConfig *config.Config) (storage.St
 
 	return store, cfg, nil
 }
+
+// injectOutputOverrides takes local configuration for specific outputs and applies them to the configuration.
+//
+// The name of the output must match or no options will be overwritten.
+func injectOutputOverrides(log *logger.Logger, rawConfig *config.Config) func(change coordinator.ConfigChange) coordinator.ConfigChange {
```

Comment:
if we indeed patch an output, could we also log it with info level just to make it easier to capture this event from agent-logs? Also, although I think this is happening already just to be safe I am gonna ask; we are gonna get the patched output in the agent diagnostics right? 🙂